src/LiveComponent/src/Metadata/LiveComponentMetadataFactory.php

```diff
@@ -130,6 +130,17 @@ private static function propertiesFor(\ReflectionClass $class): iterable
         }
     }
 
+    private function createQueryStringMapping(string $propertyName, LiveProp $liveProp): array
+    {
+        if (false === $liveProp->url()) {
+            return [];
+        }
+
+        return [
+            'name' => $propertyName,
```

Comment:
Do we want to use "fieldName" there ?

Reply:
Not sure, this is the parameter name used in the URL (currently defaults to the property name). `fieldName` would rather relate to the property name, no?

Reply:
Ow I think I misunderstood your initial remark, you meant that we should prefer the `fieldName` value of the prop instead of the `propertyName`?

Reply:
Yeah that was my (badly expressed 😅 ) idea

Reply:
Ok so yes I agree it should default to the field name

Reply:
That was a good suggestion, this use case doesn't work properly. I'll do some refactoring here, because the mapping is not consistent with the field name.